action.yml

```diff
@@ -21,6 +21,10 @@ inputs:
     description: "Wether the coverage check should exit."
     required: false
     default: "true"
+  metric:
+    description: "The metric to get and process."
+    required: true
```

Comment:
This should be false, it already has a (backward compatible) default value.

Reply:
Right. It it redundant/useless to use `required` when using `default`.

Reply:
Done